### ISSUE TITLE
Problem with from .import db

### ISSUE THREAD

**Issue by quirosaspron** [techwithtim/Flask-Web-App-Tutorial#14]
In models.py, when I run from . import db, I get the error ImportError: attempted relative import with no known parent package. 
Both __init__.py and models.py are inside the folder website. I can't find the solution anywhere, I've tried several things but none work. 
Please help!

**Comment by abhikush** [comment]
I get the same error.

**Comment by elpython3** [comment]
Yes, you are not supposed to run anything with a `from . import <insert here>`.
